brief_self_control_scale_survey/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Brief Self-Control Scale Survey",
```

Comment:
This survey is already present in the surveys repository. https://github.com/expfactory/expfactory-surveys/tree/master/brief_self_control_survey and in expfactory-experiments https://github.com/expfactory-experiments/brief-self-control-survey.

Reply:
I didn't notice that.